avaframe/com4FlowPy/flowCore.py

```diff
@@ -191,11 +190,14 @@ def run(optTuple):
 
     if varUmaxBool:
         varUmaxArray = np.load(tempDir / ("varUmax_%s_%s.npy" % (optTuple[0], optTuple[1])))
-        if optTuple[2]["varUmaxType"].lower() == 'umax':
-            print(optTuple[2]["varUmaxType"].lower())
-            varUmaxArray = varUmaxArray**2 / 2 / 9.81
-        elif optTuple[2]["varUmaxType"].lower() != 'zdeltalim':
-            log.error("PLease provide the type of the uMax Limit: 'uMax' (in m/s) or zDeltaMax (in m)!")
+        if np.any(varUmaxArray[release>0] > 0):
```

Comment:
if ```np.any(varUmaxArray[release>0] >0``` is used here, there might still be values in ```varUmaxArray[release>0]``` which are smaller or equal 0! - in case of negative Values in ```varUmaxArray``` and ```"uMaxType"='umax'``` the negative values (e.g. noData values encoded as '-9999') will be converted to positive ones by ```varUmaxArray = varUmaxArray**2 / 2 / 9.81``` because of the ```**2```

Reply:
maybe change ```np.any``` to ```np.all``` (if all release cells should/must have a positive varUmax value) or make a separate check if the values are positive.